src/storage.rs

```diff
@@ -1,8 +1,42 @@
-pub struct FileStorage {}
+use miniserde::{json, Deserialize, Serialize};
+use std::fs;
+use std::path::Path;
+
+const FILENAME: &str = "filestorage.json";
+
+#[derive(Serialize, Deserialize, Debug)]
+pub struct FileStorage {
+    username: String,
+    password: String,
+    token: String,
```

Comment:
Instead of denoting the empty/unset value as an empty string, should we use `Option<String>`? It might be clearer in other code to differentiate between `None` and `Some("api_token")`.